docs/book/src/providers.md

```diff
@@ -37,3 +37,12 @@ To implement a secrets-store-csi-driver provider, you can develop a new provider
 - Provider mounts `<kubelet root dir>/pods` (default: [`/var/lib/kubelet/pods`](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/v0.0.14/deploy/secrets-store-csi-driver.yaml#L86-L87)) with [`HostToContainer` mount propagation](https://kubernetes-csi.github.io/docs/deploying.html#driver-volume-mounts) to be able to write the external secrets store content to the volume target path
 
 See [design doc](https://docs.google.com/document/d/10-RHUJGM0oMN88AZNxjOmGz0NsWAvOYrWUEV-FbLWyw/edit?usp=sharing) for more details.
+
+## Features supported by current providers
+
+| Features \ Providers               | Azure | GCP   | AWS   | Vault |
+| ---------------------------------- | ----- | ----- | ----- | ----- |
+| Sync as Kubernetes secret          | Yes   | Yes   | Yes   | Yes   |
+| Rotation                           | Yes   | No    | Yes   | No    |
+| Windows                            | Yes   | No    | No    | No    |
+| Service account volume projection  | No    | Yes   | Yes   | Yes   |
```

Comment:
Can we please valid for each provider?

@aramase @tam7t @lasred @tomhjp

Reply:
I'm not quite sure what "Service account volume projection" is, do we have a features doc page for that?

Reply:
We are referring to Workload Identity here. I don't believe we have a page for this.

Reply:
I think this line item should be removed. We can maybe add something like it back for support for [KEP 1855](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1855-csi-driver-service-account-token/README.md)

Reply:
@aramase  thoughts?

Reply:
+1 for @tam7t's suggestion. We can add it when we switch to `RequiresRepublish` and support service account tokens for rotation.

Reply:
updated